template_handler/cdk.py

```diff
@@ -0,0 +1,199 @@
+import aws_cdk
+import importlib.machinery
+import importlib.util
+import os
+import posixpath
+import shutil
+import subprocess
+import yaml
+from botocore.credentials import Credentials
+from pathlib import Path
+from sceptre import exceptions
+from sceptre.template_handlers import TemplateHandler
+from typing import Dict
+
+
+class CDK(TemplateHandler):
+    """
+    A template handler for AWS CDK templates. Using this will allow Sceptre to work with the AWS CDK to
+    build and package a CDK template and deploy it with Sceptre.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(CDK, self).__init__(*args, **kwargs)
```

Comment:
This `super()` syntax isn't necessary any more since Python 2 is no longer supported.
```suggestion
        super().__init__(*args, **kwargs)
```

Reply:
Also, if you're not overriding anything in `__init__`, you don't really need to define it.

Reply:
This was cloned from https://github.com/Sceptre/sceptre-template-handler-template/blob/main/template_handler/custom_template_handler.py

Reply:
That may be, but it doesn't mean you should retain it if you don't have a need for it here.